DevTools/ClientHints/explainer.md

```diff
@@ -0,0 +1,53 @@
+# Edit Client Hint headers 
+
+Authors: [Erica Draud](https://github.com/erdraud)
+
+## Status of this Document
+* This document status: **Active**
+* **Current version: this document**
+    
+## Introduction
+
+[User-Agent Client Hints](https://wicg.github.io/ua-client-hints/) are a new expansion to the Client Hints API, that enables developers to access information about a user's browser in a privacy-preserving and ergonomic way. The existing edit and send experience for these client hints has been described as limited and “super painful” by our internal clients. This is a high priority for the Web Platform team; they want to encourage people to switch to using Client Hints over Strings as we hypothesize it will greatly improve site compatibility and code hygiene. They’ve gotten feedback that limited testing infrastructure is a large friction point in that migration, so we are taking on this work to support developers and to improve our existing UX.  
```

Comment:
Regarding the "edit and send experience": would you mind explaining what the experience is like now? In what way is it limiting and painful?

Reply:
Added more context/clarification